From 9fd3825218abd28895e01bed4d8f4ea7cbaa902c Mon Sep 17 00:00:00 2001
From: Wei-Chiu Chuang <weichiu@apache.org>
Date: Thu, 25 Aug 2016 09:49:23 -0700
Subject: [PATCH 1794/2748] MAPREDUCE-6764. Teragen LOG initialization bug.
 Contributed by Yufei Gu.

(cherry picked from commit 1360bd2d545134b582e70f2add33a105710dc80b)

Change-Id: I421c10935fdbff5d6dc66d46219058983550ba88
---
 .../apache/hadoop/examples/terasort/TeraGen.java   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-examples/src/main/java/org/apache/hadoop/examples/terasort/TeraGen.java b/hadoop-mapreduce-project/hadoop-mapreduce-examples/src/main/java/org/apache/hadoop/examples/terasort/TeraGen.java
index 7e67934..745a49f 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-examples/src/main/java/org/apache/hadoop/examples/terasort/TeraGen.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-examples/src/main/java/org/apache/hadoop/examples/terasort/TeraGen.java
@@ -66,7 +66,7 @@
  * <b>bin/hadoop jar hadoop-*-examples.jar teragen 10000000000 in-dir</b>
  */
 public class TeraGen extends Configured implements Tool {
-  private static final Log LOG = LogFactory.getLog(TeraSort.class);
+  private static final Log LOG = LogFactory.getLog(TeraGen.class);
 
   public static enum Counters {CHECKSUM}
 
-- 
1.7.9.5

